app/controllers/rmi_users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class RmiUsersController < ApplicationController
+  def show
+    @rmi_user = RmiUser.find(params[:id])
+    if rmi_user_signed_in? && current_rmi_user.id == @rmi_user.id
+      @portfolios = Portfolios.all
```

Comment:
- [ ] can you redirect to the rmi_user show view instead? I realized because the rmi user can also edit the building types, we probably should allow them to navigate to either all the portfolios or to editing/creating building type questionnaires